README.md

```diff
@@ -38,7 +38,9 @@ unmarkComment: false
 # Comment to post when closing a stale Issue or Pull Request. Set to `false` to disable
 closeComment: false
 # Limit to only `issues` or `pulls`
-# only: issues
+only: issues
+# Lock stale issues
+lock: false
```

Comment:
I wonder if this should be disabled by default (commented out) too. Is locking closed issues the ideal behavior for most projects?

Reply:
Wouldn't setting the default to `false` do the same thing as commenting it out?